crates/rpc/rpc-eth-types/src/simulate.rs

```diff
@@ -108,7 +111,7 @@ where
 ///
 /// If validation is enabled, the function will return error if any of the transactions can't be
 /// built right away.
-pub fn resolve_transaction<DB: Database, Tx, T: TransactionCompat<Tx>>(
+pub fn resolve_transaction<DB: Database, Tx, N, T: TransactionCompat<N>>(
```

Comment:
let's move the trait bounds down to the where clause along with the other trait bounds